be/src/storage/schema_change_utils.cpp

```diff
@@ -706,6 +706,10 @@ Status SchemaChangeUtils::parse_request_normal(const TabletSchemaCSPtr& base_sch
                        base_schema->has_index(ref_column.unique_id(), GIN)) {
                 *sc_directly = true;
                 return Status::OK();
+            } else if (new_schema->has_index(new_column.unique_id(), NGRAMBF) !=
+                       base_schema->has_index(ref_column.unique_id(), NGRAMBF)) {
+                *sc_directly = true;
```

Comment:
do we need to consider the NgramBfIndexParams? such as gram num or bloom filter's false positive possibility

Reply:
right now ngram bloom filter only support add index and drop index, one column only have at most one ngram bloom filter index, so if want to modify gram num or fpp, user need to drop existed index and create new index